@kiva/kv-loan-filters/src/tests/distributionModels.spec.ts

```diff
@@ -115,6 +115,11 @@ describe('distributionModels.ts', () => {
 				expect(distributionModels.getFlssFilter({ distributionModel: 'DIRECT' }))
 					.toEqual({ distributionModel: { eq: 'DIRECT' } });
 			});
+
+			it('should capitalize filters', () => {
+				expect(distributionModels.getFlssFilter({ distributionModel: 'direct' }))
+					.toEqual({ distributionModel: { eq: 'DIRECT' } });
+			});
```

Comment:
This test failed with the current state and was resolved with the updates in the module.